### PR TITLE
In-meeting notifications nodejs sample

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

File: samples/meetings-notification/nodejs/README.md [hunk 1]
```diff
@@ -10,16 +10,16 @@ languages:
 extensions:
  contentType: samples
  createdDate: "07/07/2021 13:38:26 PM"
-urlFragment: officedev-microsoft-teams-samples-meetings-content-bubble-nodejs
+urlFragment: officedev-microsoft-teams-samples-meetings-notification-nodejs
 ---
 
-# Meetings Content Bubble
+# Targeted In-Meeting Notification
```
Comment (ChetanSharma-msft) [inline samples/meetings-notification/nodejs/README.md:16]: Add `##` Targeted In-Meeting Notification

File: samples/meetings-notification/nodejs/README.md [hunk 2]
```diff
@@ -10,16 +10,16 @@ languages:
...
 
-This sample illustrates how to implement [Content Bubble](https://docs.microsoft.com/microsoftteams/platform/apps-in-teams-meetings/create-apps-for-teams-meetings?view=msteams-client-js-latest&tabs=dotnet#notificationsignal-api) In-Meeting Experience.
+This sample illustrates how to implement [Targeted In-Meeting Notification](https://review.learn.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/meeting-apps-apis?branch=pr-en-us-7615&tabs=dotnet#targeted-meeting-notification-api)  In-Meeting Experience.
```
Comment (ChetanSharma-msft) [inline samples/meetings-notification/nodejs/README.md:18]: Remove en-us

File: samples/meetings-notification/nodejs/bots/targetedMeetingNotificationBot.js [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+      }
+      else if (context.activity.value.Type == "SendTargetedMeetingNotification") {
+        var adaptiveCardChoiceSet = context.activity.value.Choice;
+        var selectedMembers = adaptiveCardChoiceSet.split(",");
+        this.targetedNotification(context, meetingId, selectedMembers);
+      }
+      else {
+        var json = JSON.stringify(context.activity.value);
+        var out = JSON.parse(json);
+        if (out.action == 'inputselector') {
```
Comment (ChetanSharma-msft) [inline samples/meetings-notification/nodejs/bots/targetedMeetingNotificationBot.js:55]: One line space before it.

File: samples/meetings-notification/nodejs/bots/targetedMeetingNotificationBot.js [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+          contentBubbleTitles.contentQuestion = out.myReview;
+          await this.contentBubble(context);
+          await context.sendActivity({ attachments: [this.createQuestionAdaptiveCard(out.myReview)] });
+        } else {
+          await context.sendActivity(context.activity.from.name + " : " + "**" + out.myReview + "**" + " for " + "'" + out.action + "'");
+        }
+      }
+      await next();
```
Comment (ChetanSharma-msft) [inline samples/meetings-notification/nodejs/bots/targetedMeetingNotificationBot.js:63]: One line space before it.

File: samples/meetings-notification/nodejs/bots/targetedMeetingNotificationBot.js [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+  // Create adaptive card for send list of agenda questions.
+  createQuestionAdaptiveCard(myText) {
+    var templatePayload = templateJson;
+    var template = new ACData.Template(templatePayload);
+    var cardPayload = template.expand({
+      $root: {
+        name: myText
+      }
+    });
+    return CardFactory.adaptiveCard(cardPayload);
```
Comment (ChetanSharma-msft) [inline samples/meetings-notification/nodejs/bots/targetedMeetingNotificationBot.js:150]: One line space before it.

File: samples/meetings-notification/nodejs/bots/targetedMeetingNotificationBot.js [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+  // Create adaptive card for send list of in-meeting participants.
+  createMembersAdaptiveCard(members) {
+    var templatePayload = notificationCardJson;
+    var template = new ACData.Template(templatePayload);
+    var cardPayload = template.expand({
+      $root: {
+        members: members
+      }
+    });
+    return CardFactory.adaptiveCard(cardPayload);
```
Comment (ChetanSharma-msft) [inline samples/meetings-notification/nodejs/bots/targetedMeetingNotificationBot.js:162]: One line space before it.

File: samples/meetings-notification/nodejs/teamsAppManifest/manifest.json [hunk 7]
```diff
@@ -1,43 +1,76 @@
...
-    "short": "Content Bubble",
-    "full": "Meeting Extensibility in Microsoft Teams"
+    "short": "Targeted In-Meeting Notification",
+    "full": "Targeted Meeting Notification Demo"
   },
   "description": {
-    "short": "Content Bubble Scenario",
-    "full": "Content Bubble Scenario Test"
+    "short": "Targeted In-Meeting Notification demo",
+    "full": " Sample app to demo how to send in meeting targeted notifications."
```
Comment (ChetanSharma-msft) [inline samples/meetings-notification/nodejs/teamsAppManifest/manifest.json:23]: This sample app shows how to send in meeting targeted notifications.

File: samples/meetings-notification/nodejs/.env [hunk 8]
```diff
@@ -1,3 +1,3 @@
-MicrosoftAppId=
-MicrosoftAppPassword=
-BaseUrl=
\ No newline at end of file
+MicrosoftAppId=<<MICROSOFT_APP_ID>>
+MicrosoftAppPassword=<<MICROSOFT_APP_PASSWORD>>
+BaseUrl=<<BaseURL>>
```
Comment (ChetanSharma-msft) [inline samples/meetings-notification/nodejs/.env:3]: Kindly run build pipeline and share the link as well.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comments.

File: samples/meetings-notification/nodejs/bots/targetedMeetingNotificationBot.js (hunk 5 above)
Comment (Prithvi-MSFT) [reply samples/meetings-notification/nodejs/bots/targetedMeetingNotificationBot.js:154]: This sample app shows how to send in meeting targeted notifications.

File: samples/meetings-notification/nodejs/bots/targetedMeetingNotificationBot.js (hunk 4 above)
Comment (Prithvi-MSFT) [reply samples/meetings-notification/nodejs/bots/targetedMeetingNotificationBot.js:65]: Added

File: samples/meetings-notification/nodejs/bots/targetedMeetingNotificationBot.js (hunk 3 above)
Comment (Prithvi-MSFT) [reply samples/meetings-notification/nodejs/bots/targetedMeetingNotificationBot.js:56]: Added

File: samples/meetings-notification/nodejs/teamsAppManifest/manifest.json (hunk 7 above)
Comment (Prithvi-MSFT) [reply samples/meetings-notification/nodejs/teamsAppManifest/manifest.json]: Done

File: samples/meetings-notification/nodejs/README.md (hunk 1 above)
Comment (Prithvi-MSFT) [reply samples/meetings-notification/nodejs/README.md]: Done

File: samples/meetings-notification/nodejs/.env (hunk 8 above)
Comment (Prithvi-MSFT) [reply samples/meetings-notification/nodejs/.env:3]: Attach build pipeline link

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**